game.go

```diff
@@ -770,18 +771,33 @@ func SchedNow() int {
 	return 0
 }
 
+func runMain(sprite Sprite) {
```

Comment:
Apart from `Main` of sprite, it is better to also track the time cost for `Main` of game?